dell/xps/15-9500/nvidia/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib, pkgs, ... }:
+
+# This creates a new 'nvidia-offload' program that runs the application passed to it on the GPU
+let
+  nvidia-offload = pkgs.writeShellScriptBin "nvidia-offload" ''
+    export __NV_PRIME_RENDER_OFFLOAD=1
+    export __NV_PRIME_RENDER_OFFLOAD_PROVIDER=NVIDIA-G0
```

Comment:
Is this specific to the hardware or general applicable for NVIDIA?

Reply:
This should be generally applicable for prime offload. It is currently only supported in unstable according to the wiki (in particular https://github.com/NixOS/nixpkgs/pull/66601).